server/WebAPI/Controllers/AuthController.cs

```diff
@@ -32,5 +32,21 @@ public IActionResult Auth(AuthRequest authRequest)
             var token = _authService.MakeToken(user);
             return Ok(new AuthResponse { AccessToken = token });
         }
+
+        [AllowAnonymous]
+        [HttpPost]
+        [Route("SignUp")]
+        public IActionResult Registration(AuthRequest authRequest)
+        {
+            if (_userService.FindUser(authRequest.Email))
```

Comment:
does find method return a user or true/false? if true/false - it is incorrect name, because find method should return object or null. for boolean it is more correct name like IsUserExist. so, it should be changed return value or name for this method